src/utils/notifications.ts

```diff
@@ -95,14 +98,10 @@ export const handleTurnOn = async (accountId: string, hideModal: () => void) =>
 };
 
 export const handleOnCancel = () => {
-  localStorage.setItem(
-    NOTIFICATIONS_STORAGE,
-    JSON.stringify({
-      ...getNotificationLocalStorage(),
-      showOnTS: Date.now() + 86400000, // 14 days
```

Comment:
@marcinbodnar  If I'm following this correctly, the removal of this may cause the `subscribe to notifications` prompt to always display. as per https://github.com/near/near-discovery/pull/551/files#diff-18e0d4553c97cfc420e938ccafb4e3a688e782fa4512ba4ceae3e2a6f24c1987R59

Reply:
ah I see. it's been moved .